### ISSUE TITLE
[tree] fix the filenames

### ISSUE THREAD

**Issue by ebzzry** [vedatechnologiesinc/vedatechnologiesinc.github.io#93]
There are files with names like `Ts.svg` and `rhaskell.svg`. We can’t have those.

<img width="1088" height="396" alt="Image" src="https://github.com/user-attachments/assets/35051619-10cb-4fe4-b68d-908a025cb811" />

The alt text for `darwin.svg` should be `Darwin` not `DarwinOS_`

The alt text for `typescript.svg` should be `TypeScript` not `Typescript_`

The alt text for `arch.svg` should be `Arch Linux`.

Remove the links for YouTube, for the mean time.
